scripts/dlv/juju.sh

```diff
@@ -71,6 +71,7 @@ fi
 
 # find out the remote host to ssh to
 remote_host=$(juju status -m ${target_model} --format=json | jq -r ".machines[\"${target_machine}\"][\"ip-addresses\"][0]")
+remote_host=${remote_host%%/*}
```

Comment:
This change makes me think that the juju status output is incorrect. That this is the wrong place to be making this change.